testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/policyHelper.java

```diff
@@ -0,0 +1,36 @@
+package com.microsoft.identity.internal.testutils.labutils;
+
+import com.microsoft.identity.internal.test.labapi.ApiException;
+import com.microsoft.identity.internal.test.labapi.api.DisablePolicyApi;
+import com.microsoft.identity.internal.test.labapi.api.EnablePolicyApi;
+import com.microsoft.identity.internal.test.labapi.model.CustomSuccessResponse;
+
+public class policyHelper {
+    public boolean enablePolicy(String upn, String policy)
+    {
+        LabAuthenticationHelper.getInstance().setupApiClientWithAccessToken();
+        final EnablePolicyApi enablePolicyApi = new EnablePolicyApi();
+        try{
+            final CustomSuccessResponse customSuccessResponse;
+            customSuccessResponse = enablePolicyApi.putPolicy(upn, policy);
+            final String expectedResult = ("Enabled : " + policy + "policy for user : " + upn)
+                    .toLowerCase();
+            return customSuccessResponse.getResult().toLowerCase().contains(expectedResult);
+        } catch (ApiException e) {
+            throw new RuntimeException(e.getMessage());
+        }
+    }
+    public boolean disablePolicy(String upn, String policy){
+        LabAuthenticationHelper.getInstance().setupApiClientWithAccessToken();
+        final DisablePolicyApi disablePolicyApi = new DisablePolicyApi();
+        try{
+            final CustomSuccessResponse customSuccessResponse;
+            customSuccessResponse = disablePolicyApi.put(upn, policy);
+            final String expectedResult = ("disabled : " + policy + " policy for user : " + upn)
+                    .toLowerCase();
+            return customSuccessResponse.getResult().toLowerCase().contains(expectedResult);
```

Comment:
Are you sure this result can never be null?

Although this is kind of a trivial example, as a matter of good practice, split multiple method calls into multiple lines. It makes debugging stack traces substantially easier.

So rather than....
```java
return customSuccessResponse.getResult().toLowerCase().contains(expectedResult);
```
do
```java
final String result  = successResponse.getResult();

if (null != result) {
    return result.toLowerCase().contains(expectedResult);
}
```

Reply:
and if it is `null`, you can throw `new AssertionError()` -- could probably pass a message in there

Reply:
modified the code to handle null results.